internal/components/receivers_test.go

```diff
@@ -127,6 +127,10 @@ func TestDefaultReceivers(t *testing.T) {
 		{
 			receiver: "googlecloudspanner",
 		},
+		{
+			receiver:     "googlecloudpubsub",
+			skipLifecyle: true,
```

Comment:
nit: Skipped tests in this file typically have a brief comment explaining why. Mind adding one here?

Reply:
@djaglowski  Added comment explaining that you need a valid pubsub subscription.